From be5e7380ae69a2620ecf488250ce7e48d51c6338 Mon Sep 17 00:00:00 2001
From: itzexor <itzexor@gmail.com>
Date: Fri, 20 Apr 2018 20:01:56 -0700
Subject: [PATCH] build: fix PKGDATADIR define / use only one define

we need to prepend the prefix to get a full path, and there's
no reason to have 3 names for the same thing.
---
 cinnamon-session/csm-inhibit-dialog.c | 2 +-
 meson.build                           | 6 +-----
 2 files changed, 2 insertions(+), 6 deletions(-)

diff --git a/cinnamon-session/csm-inhibit-dialog.c b/cinnamon-session/csm-inhibit-dialog.c
index 15d2f8d..aec31e4 100644
--- a/cinnamon-session/csm-inhibit-dialog.c
+++ b/cinnamon-session/csm-inhibit-dialog.c
@@ -1075,7 +1075,7 @@ csm_inhibit_dialog_init (CsmInhibitDialog *dialog)
 
         error = NULL;
         if (!gtk_builder_add_from_file (dialog->priv->xml,
-                                        GTKBUILDER_DIR "/" GTKBUILDER_FILE,
+                                        PKGDATADIR "/" GTKBUILDER_FILE,
                                         &error)) {
                 if (error) {
                         g_warning ("Could not load inhibitor UI file: %s",
diff --git a/meson.build b/meson.build
index 4761c6e..b56c544 100644
--- a/meson.build
+++ b/meson.build
@@ -143,11 +143,7 @@ conf.set('ENABLE_IPV6', have_ipv6)
 
 rootInclude = include_directories('.')
 
-pkgdatadir = join_paths(get_option('datadir'), meson.project_name())
-
-conf.set_quoted('GTKBUILDER_DIR',     pkgdatadir)
-conf.set_quoted('DATA_DIR',           pkgdatadir)
-conf.set_quoted('PKGDATADIR',         pkgdatadir)
+conf.set_quoted('PKGDATADIR',         join_paths(get_option('prefix'), get_option('datadir'), meson.project_name()))
 conf.set_quoted('LIBEXECDIR',         get_option('libexecdir'))
 conf.set_quoted('LOCALE_DIR',         join_paths(get_option('prefix'), get_option('localedir')))
 
